performance/README.md

```diff
@@ -0,0 +1,135 @@
+# neo4j-streams performance test
+
+This part of the project aims to give a standard way to understand the performances of the platform and highlight the configuration results.
+
+
+
+Currently, it's available only the test of neo4j to neo4j configuration, in remote too.
+
+## Setup
+
+The basic unit of work used to test the performances needs a specific cypher part in order to get the timestamp of creation and receiving of the event. The elapsed time is calculated using these timestamps through cypher queries.
+
+### Producer
+
+- neo4j 3.4.7 community edition
+- APOC 3.4.0.3
```

Comment:
which kafka version?
we probably shouldn't need apoc.

Reply:
Updated the PR with the kafka version. The apoc is used to put the timestamp into the nodes created. Using the apoc.date.currentTimestamp() each node has a different timestamp in the same transaction, so I can pick the min in the producer and the max in the consumer